.github/workflows/merge-queue.yml

```
@@ -24,175 +24,177 @@ env:
+  #       id: get-cache
+  #       uses: ./.github/actions/get-cache
+  #       with:
+  #         github-token: ${{ secrets.GITHUB_TOKEN }}
+  #         keys: ${{ needs.prepare.outputs.CACHE_KEYS }}
+  #         enable-cache: 'node_modules,generated-files'
+
+  #     - name: Building
+  #       run: ./scripts/ci/run-in-parallel-native.sh build
 
   success:
     runs-on: arc-runners
```

actionlint (1.7.4)

label "arc-runners" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-22.04", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "macos-12-xl", "macos-12-xlarge", "macos-12-large", "macos-12", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)